### ISSUE TITLE
Can't resolve all parameters for ScrollHooks

### ISSUE THREAD

**Issue by Tristen1099** [tjoskar/ng-lazyload-image#476]
I'm getting the error ERROR in Can't resolve all parameters for ScrollHooks in C:/.........../node_modules/ng-lazyload-image/ng-lazyload-image.d.ts: (?). This only happens when doing ng build --prod but works fine with ng build. I have tries npm install ng-lazyload-image --save-prod with no success.

**Comment by ketanakbari** [comment]
i also facing this kind of issue while using `--prod` in build command.

**Comment by tjoskar** [comment]
What version of Angular are you using?
Can you place the full error message?

**Comment by Tristen1099** [comment]
Angular 8.3
ERROR in Can't resolve all parameters for ScrollHooks in C:........../node_modules/ng-lazyload-image/ng-lazyload-image.d.ts: (?).

**Comment by tjoskar** [comment]
Are you using the ScrollHook? Or how does your setup look like?

**Comment by tjoskar** [comment]
Does it look someting like:
```ts
import { NgModule } from '@angular/core';
import { BrowserModule } from '@angular/platform-browser';
import { LazyLoadImageModule, LAZYLOAD_IMAGE_HOOKS, ScrollHooks } from 'ng-lazyload-image'; // <-- include ScrollHooks
import { AppComponent } from './app.component';

@NgModule({
  declarations: [AppComponent],
  imports: [BrowserModule, LazyLoadImageModule],
  providers: [{ provide: LAZYLOAD_IMAGE_HOOKS, useClass: ScrollHooks }],
  bootstrap: [AppComponent],
})
export class MyAppModule {}
```

**Comment by Tristen1099** [comment]
Yes, the imports are exactly like that in the App Module class.

**Comment by tjoskar** [comment]
A fix is included in `ng-lazyload-image@9.0.1` so you don't need to create an extra empty class.
